_events/sr2025/kickstart.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Kickstart
+date: 2024-10-12 11:00:00 Europe/London
+end_date: 2024-10-12 17:00:00 Europe/London
+layout: event
+type: kickstart
+location: Southampton University
+---
+
+Kickstart is the event which kicks off a competition year. The structure of the
+competition and the game rules will be announced and of course any questions you
+have will be answered.
+
+Throughout the day we'll have Blueshirts (our awesome volunteers!) around in
+person, ready to answer any questions you may have and to help you through the
+microgames. These microgames will help your team become familiar with the Kit
+you've been provided and should kickstart some ideas for your robot's design.
+
+If you are unable to attend Kickstart, please [let us know][teams-contact].
```

Comment:
We probably want to mention something about streaming it

Reply:
4ea43de106f420259bf24576ded9d86610ad71da